GeneratorInterface/Core/src/EmbeddingHepMCFilter.cc

```diff
@@ -4,7 +4,8 @@
 #include "boost/algorithm/string/trim_all.hpp"
 
 EmbeddingHepMCFilter::EmbeddingHepMCFilter(const edm::ParameterSet &iConfig)
-    : ZPDGID_(iConfig.getParameter<int>("BosonPDGID")), includeDY_(iConfig.getParameter<bool>("IncludeDY")) {
+    : ZPDGID_(iConfig.getParameter<int>("BosonPDGID")),
+      includeDY_(iConfig.getUntrackedParameter<bool>("IncludeDY", false)) {
```

Comment:
And, by the way (sorry if I am slow....): why untracked? It affects the output as the other config parameters do, and it should be tracked then?

Reply:
How do I correctly make it a tracked, but optional parameter with a default value?

Reply:
Just like this?
```
EmbeddingHepMCFilter::EmbeddingHepMCFilter(const edm::ParameterSet &iConfig)
  : ZPDGID_(iConfig.getParameter<int>("BosonPDGID")) {
    if (ps.exists("includeDY")) {
          includeDY_ = iConfig.getParameter<std::string>("IncludeDY");
    }
```
The default of `includeDY_` is already set to `false` in the header file.